target/linux/ipq806x/image/Makefile

```diff
@@ -189,75 +199,82 @@ define Device/netgear_r7800
 	PAGESIZE := 2048
 	BOARD_NAME := r7800
 	SUPPORTED_DEVICES += r7800
-	DEVICE_TITLE := Netgear Nighthawk X4S R7800
 	DEVICE_PACKAGES := ath10k-firmware-qca9984-ct
 endef
 TARGET_DEVICES += netgear_r7800
 
 define Device/qcom_ipq8064-ap148
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
+	DEVICE_VENDOR := Qualcomm
+	DEVICE_MODEL := AP148
 	DEVICE_DTS := qcom-ipq8064-ap148
 	KERNEL_INSTALL := 1
 	BLOCKSIZE := 128k
 	PAGESIZE := 2048
 	BOARD_NAME := ap148
 	SUPPORTED_DEVICES += ap148
-	DEVICE_TITLE := Qualcomm AP148
 	DEVICE_PACKAGES := ath10k-firmware-qca99x0-ct
 endef
 TARGET_DEVICES += qcom_ipq8064-ap148
 
 define Device/qcom_ipq8064-ap148-legacy
 	$(call Device/LegacyImage)
 	$(call Device/UbiFit)
+	DEVICE_VENDOR := Qualcomm
+	DEVICE_MODEL := AP148
+	DEVICE_VARIANT := (legacy)
```

Comment:
Brackets or not ...?
Despite, if that's variant "legacy", we need to also provide a variant for the "standard" one.